public/index.html

```diff
@@ -58,7 +58,7 @@ <h3 class="ml-1 fontHeading">Create New Task</h3>
                         <hr class="w-75 ml-0">
                     </div>
                     <div class="alert bg-light form-inline w-50 col-md-3 mb-5" style="font-size: 20px;" role="alert">
-                        <font style="font-weight: bold;" class="mr-3">Date:</font>
+                        <div style="font-weight: bold;" class="mr-3">Date:</div>
```

Comment:
Please convert all the margins into padding

Reply:
okay